libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -57,6 +57,9 @@ export interface IAppHeaderProps {
 
     showUpsellButton?: boolean;
     onUpsellButtonClick?: (e: React.MouseEvent) => void;
+
+    showInviteButton?: boolean;
+    onInviteButtonClick?: (e: React.MouseEvent) => void;
```

Comment:
Please rename `onInviteButtonClick` → `onInviteItemClick`, as the item is not a button, is it?